lib/project_types/theme/commands/deploy.rb

```diff
@@ -14,7 +15,16 @@ def call(*)
             @ctx.abort(@ctx.message('theme.deploy.abort'))
           end
 
-          unless Themekit.deploy(@ctx, env: options.flags[:env])
+          if options.flags[:env]
+            env = options.flags[:env]
+            options.flags.delete(:env)
+          end
+
+          flags = options.flags.map do |key, _value|
```

Comment:
I assume it would be possible for flags to have values, even though we currently don't use any. Ideally, we could make sure to handle `value` here appropriately.